src/PasswordField.js

```diff
@@ -20,15 +20,12 @@ const getStyles = (props, context, state) => {
 
   const styles = {
     root: {
-      display: 'flex',
-      width: props.fullWidth ? '100%' : null
-    },
-    inputContainer: {
-      flex: 1
+      position: 'relative',
```

Comment:
Okay, then maybe we should keep it as it is.
I just saw that `fullWidth` doesn't set the `width` to 100% anymore, so the component won't actually get the full width.